modyn/storage/src/CMakeLists.txt

```diff
@@ -68,51 +68,10 @@ message(STATUS "Proto generated files in ${PROTO_BINARY_DIR}: ${PROTO_GENERATED_
 
 target_include_directories(modyn-storage-proto PUBLIC "$<BUILD_INTERFACE:${PROTO_BINARY_DIR}>")
 
-target_link_libraries(modyn-storage-proto PUBLIC grpc++ grpc++_reflection)
-
 if (MODYN_USES_LOCAL_GRPC)
-    message(STATUS "Since we are using local gRPC, we need to fix linking issues. If you encounter errors, consider building gRPC from source.")
```

Comment:
Hmh, this was necessary at the time (at least on my mac using gRPC installed via brew), but apparently, all tests run through without it.